packages/connect-voting-disputable/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": "src"
+  },
+  "include": ["./src/**/*.ts"],
```

It seems that specifying the directory only is enough (I moved the other packages to do the same):

```suggestion
  "include": ["src"],
```